Esri/arcgis-maps-sdk-swift-samples
# [Update] Improve visibility of dynamic entities in sample

## Description

This PR addresses specific feedback from Nick on the initial testing for 200.1.0 sample viewer TestFlight app to improve the visibility of the dynamic entities in the `Add dynamic entity layer` sample. The visibility of the dynamic entities is improved by adding content offsets to the MapView so that the dynamic entities are still visible when the sheet is open.

>- [ ] Add dynamic entity layer
>> Can the sheet be made to cover less of the screen? Or praise the visible area offsets could be used to keep the dynamic entities in view while the sheet is open?

## Linked Issue(s)

- `swift/issues/3931`

## How To Test
- Toggle the `Dynamic Entity Settings` sheet and verify that there are dynamic entities in view.
- Interact with the sheet and observe changes to the dynamic entities.

## Screenshots
Before/After


<details>
  <summary>iPhone 14 Pro Max</summary>
   <img src="https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/117859673/2b07bbf6-fc75-4f3a-aa61-9afd993a31ab" width="300">    <img src="https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/117859673/03e44b57-5660-4263-ace3-09fabfe7bbb5" width="300">
  </details>
  
 <details>
  <summary>iPad Pro</summary>
   <img src="https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/117859673/0415936f-3eae-4002-8550-adeedb72b35f" width="400">    <img src="https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/117859673/8bca9664-62af-4181-84e0-61e53fab5cfe" width="400">
  </details>
  
   <details>
  <summary>Mac Catalyst</summary>
   <img src="https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/117859673/5bfea7fd-a207-42fe-82e8-3ea862a27bcf" width="600">    <img src="https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/117859673/02d9095d-b7f6-477d-8c05-6aa7c4773ad2" width="600">
  </details>


## To Discuss

- Would a sheet that covers less of the screen be a preferred fix? A sheet with presentation detents `.presentationDetents([.fraction(0.40)])` does result in a shorter sheet on iPhone where the dynamic entities are visible, however, results in a full screen sheet on iPad and Mac Catalyst which covers the dynamic entities.
